pkg/rancher-components/src/components/Form/Radio/RadioGroup.vue

```diff
@@ -1,8 +1,14 @@
-<script>
+<script lang="ts">
+import Vue from 'vue';
 import { _VIEW } from '@shell/config/query-params';
-import { RadioButton } from '@components/Form/Radio';
+import RadioButton from '@components/Form/Radio/RadioButton.vue';
 
-export default {
+interface option {
```

Comment:
Interface always Capitalcase.

Also we do not know the type, so we say always unknown.

```ts
interface Option {
  value: unknown,
  label: string
}
```

You have also to update these props:

```ts
    options: {
      type:     Object as () => Option[],
      required: true
    },

    labels: {
      type:    Object as () => string[],
      default: null
    },```